gallery.css

```diff
@@ -0,0 +1,87 @@
+
```

Comment:
Lots of responsive units, and that's great. I do notice you used pixels in at least one place. Probably just an oversight! Otherwise consistent.